### PR TITLE
chore: update deps and webpack config

### PR DESCRIPTION
**Pull request by ThorsAngerVaNeT**
1) Update dependencies to prevent errors with old versions
2) Update webpack config
3) Delete script tag with index.js from index.html to prevent double script execution

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -17,19 +17,19 @@
...
-        "webpack-cli": "^4.5.0",
-        "webpack-dev-server": "^3.11.2",
-        "webpack-merge": "^5.7.3"
+        "clean-webpack-plugin": "^4.0.0",
+        "copy-webpack-plugin": "^12.0.2",
+        "css-loader": "^6.10.0",
+        "eslint": "^8.56.0",
+        "eslint-config-prettier": "^9.1.0",
+        "eslint-plugin-import": "^2.29.1",
+        "eslint-plugin-prettier": "^5.1.3",
```
Comment (gentoosiast) [inline package.json:27]: I think we should get rid of `eslint-plugin-prettier`, it's no longer recommended on official Prettier website

File: webpack.dev.config.js [hunk 2]
```diff
@@ -4,6 +4,6 @@ module.exports = {
     mode: 'development',
     devtool: 'inline-source-map',
     devServer: {
-        contentBase: path.resolve(__dirname, '../dist'),
+        static: path.resolve(__dirname, '../dist'),
```
Comment (gentoosiast) [inline webpack.dev.config.js]: Also it will be nice to fix problem with `dist` created one level above.

File: package.json (hunk 1 above)
Comment (ThorsAngerVaNeT) [reply package.json:27]: It's personal preference thing even if prettier not recommended it, so I believe there is always be both sides

File: webpack.dev.config.js (hunk 2 above)
Comment (ThorsAngerVaNeT) [reply webpack.dev.config.js]: Oh, right, good catch. Thank you

**Review by valerydluski — APPROVED** [review]

**Pull request merged**